### DISCUSSION TITLE
Source map geneation for exported Next app

### DISCUSSION THREAD

**Discussion by JerryPan** [vercel/next.js#13495] [Help] [answered]
Is there an option to get source maps generated for 'next export' ?    I would like to enable source map and easy debugging for exported static app that I plan to be served by a Flask server, but I found I can not easily debug and trace my JS code in such a deployment.  Any pointers appreciated.

**Comment by jamesmosier** [comment, marked as answer]
You can use [next-source-maps](https://github.com/vercel/next-plugins/tree/master/packages/next-source-maps) in your `next.config.js` to get sourcemaps

**Reply by JerryPan** [reply]
thank you, that worked nicely
